.circleci/config.yml

```
@@ -1,26 +1,514 @@
+          command: ./scripts/download-fonts
+      - run:
+          name: Build App
+          command: ./scripts/ci-ios
+      - save_cache:
+          key: v1-app_build_ios-{{ checksum ".manifests/app_build" }}
+          paths:
+            - derived_data
+            - node_modules/react-native-config
+  prep-android-env:
+    steps:
+      - attach_workspace:
+          at: ../workspace
+      - run:
+          name: Clear project dir
+          command: |
+            rm -rf /Users/distiller/project
+      - checkout
+      - attach_workspace:
+          at: .
+      - install-node
+      - setup-awscli
+      - setup-env-file
+      - restore_cache:
+          keys:
+            - v1-test-success-{{ checksum "../workspace/.manifests/android_native" }}
+      - run:
+          name: Quit early if possible
+          command: |
+            if test $CIRCLE_BRANCH != beta-android && ls build-success.log
+            then
+              circleci step halt
+            fi
+      - install-node-modules
+      - run-relay-compiler
+      - switch-ruby
+      - install-gems
+  build-app-android:
+    steps:
+      - restore_cache:
+          keys:
+            - v1-app_build_android-{{ checksum ".manifests/app_build" }}
+      - run:
+          name: Download fonts from s3
+          command: ./scripts/download-fonts
+      - run:
+          name: Build App
+          command: ./scripts/ci-android
+      - save_cache:
+          key: v1-app_build_android-{{ checksum ".manifests/app_build" }}
+          paths:
+            - android/build
+            - android/app/build
+            - node_modules/react-native-config
+
 jobs:
-  say-hello:
-    # Specify the execution environment. You can specify an image from Dockerhub or use one of our Convenience Images from CircleCI's Developer Hub.
-    # See: https://circleci.com/docs/2.0/configuration-reference/#docker-machine-macos-windows-executor
+  detect-secrets:
+    docker:
+      - image: artsy/detect-secrets:1.2.0
+    working_directory: /usr/src/app
+    steps:
+      - checkout
+      - run: ./scripts/secrets-check-all
+
+  deploy-nightly-beta:
+    environment:
+      BUNDLE_PATH: .vendor # path to install gems and use for caching
+    macos:
+      xcode: 13.2.1
+    steps:
+      - checkout
+      - install-node
+      - run:
+          name: Deploy beta
+          command: ./scripts/deploy-both
+
+  check-pr:
+    executor:
+      name: node/default
+      tag: "16.6.0"
+    steps:
+      - checkout
+      - generate-checksums
+      - install-node-modules
+      - run-relay-compiler
+      - run:
+          name: Danger
+          command: yarn danger ci --verbose
+
+  check-if-app-has-been-approved-for-release:
+    environment:
+      BUNDLE_PATH: .vendor # path to install gems and use for caching
+
+    macos:
+      xcode: 13.2.1
+
+    steps:
+      - checkout
+      - generate-checksums
+      - switch-ruby
+      - install-gems
+      - run:
+          name: Check flags
```

```suggestion
          name: Check if app has been approved for release
```
think this is wrong in Eigen as well